src/vireshJJun20/Assignment25/SumOfNumbersInString.java

```diff
@@ -0,0 +1,24 @@
+package vireshJJun20.Assignment25;
+
+public class SumOfNumbersInString {
+	int sum = 0;
+
+	void getSum(String str) {
+		for (int index = 0; index < str.length(); index++) {
+			if (Character.isDigit(str.charAt(index))) {
+				sum = sum + Integer.parseInt(String.valueOf(str.charAt(index)));
```

Comment:
You can use below method to convert char to int
Character.getNumericValue(numberString.charAt(index));